README_zh.md

```
@@ -1,13 +1,23 @@
 <picture>
-  <img alt="MineContext" src="src/MineContext-Banner.svg" width="80%" height="auto">
+  <img alt="MineContext" src="src/MineContext-Banner.svg" width="100%" height="auto">
 </picture>
 
+### MineContext：洞察本质，激发创造
+
 一个开源、主动的上下文感知AI伙伴，致力于让您的工作、学习与创作更加清晰高效。
 
 <a href="https://github.com/volcengine/MineContext/issues">反馈问题</a> · <a href="https://bytedance.larkoffice.com/share/base/form/shrcn2wgAfiyCVVwhvVYCXWNNdc">提交问卷</a>
 
+[![][release-shield]][release-link]
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)